### DISCUSSION TITLE
strapi "test" restaurant application - the documentation needs a review.  When creating the react-app for this project, it is not clear where this should sit in the file structure.

### DISCUSSION THREAD

**Discussion by SunbeamRapier** [strapi/strapi#7881] [Legacy Topics]
In the instructions for the restaurant app there are numerous minor errors and, in general, the instructions are harder to follow than they ought to be.

Having built the restaurant, I attempted to create the react-app in a subdirectory of the restaurant folder.  That resulted in a webpack error.  So where should the react-app be created?

Similarly, the instruction to add this code...

import axios drom 'axios'
axios.get('http://localhost:1337/restaurants')
  .then(response => {
    console.log(response);
  })

... is obviously intended to be added to a component in the new react-app - but which component?

**Comment by derrickmehaffy** [comment]
Can you please clarify what example you are referring to?
